.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 # next.js build output
 .next
+
+.directory
```

Comment:
This is fine, but since these `.directory` files are being created by your computer, rather than anything specific to the project, putting it in a global `.gitignore` file on your computer would make sense--otherwise any project you check out and view with your file manager will end up showing these as untracked files.